orttraining/orttraining/test/python/orttraining_test_ortmodule_deepspeed_pipeline_parallel.py

```diff
@@ -0,0 +1,119 @@
+import torch
+from torch import nn, optim
+import torch.distributed as dist
+import deepspeed
+from deepspeed.pipe import PipelineModule, LayerSpec
+from deepspeed.utils import RepeatingLoader
+
+import onnxruntime
+from onnxruntime.training import ORTModule
+
+import argparse
+
+# USAGE:
+# pip install deepspeed
+# deepspeed orttraining_test_ortmodule_deepspeed_pipeline_parallel.py --deepspeed_config=orttraining_test_ortmodule_deepspeed_pipeline_parallel_config.json --pipeline-parallel-size 2 --steps=100
+# expected output : steps: 100 loss: 0.0585 iter time (s): 0.186 samples/sec: 53.694 
+
+class SampleData(torch.utils.data.Dataset):
+    def __init__(self,x,y):
+        self.x = x
+        self.y = y
+    def __len__(self):
+        return x.size()[0]
+    def __getitem__(self, idx):
+        return self.x[idx],self.y[idx]
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--local_rank',
+                        type=int,
+                        default=-1,
+                        help='local rank passed from distributed launcher')
+    parser.add_argument('-s',
+                        '--steps',
+                        type=int,
+                        default=100,
+                        help='quit after this many steps')
+    parser.add_argument('-p',
+                        '--pipeline-parallel-size',
+                        type=int,
+                        default=2,
+                        help='pipeline parallelism')
+    parser.add_argument('--backend',
+                        type=str,
+                        default='nccl',
+                        help='distributed backend')
+    parser.add_argument('--seed', type=int, default=0, help='PRNG seed')
+    parser.add_argument('--fp16',type=bool,default=False,help='fp16 run')
+    parser.add_argument('--run_without_ort',type=bool,default=False,help='onlydeepspeed run')
+    
+    parser = deepspeed.add_config_arguments(parser)
+    args = parser.parse_args()
+    return args
+
+print('Running deepspeed pipeline parallel module with ORTModule')
```

Comment:
```suggestion
if args.run_without_ort:
   print('Running deepspeed pipeline parallel module without ORTModule')
else:
   print('Running deepspeed pipeline parallel module with ORTModule')
```
This test can be execute with and without ortmodule, right?

Reply:
done

Reply:
yes it can run without ORTModule and matches the result without ORT. Also, Addressed your suggestions. Thank you.